linux/utils/enable_vmtools_logging.yml

```diff
@@ -11,21 +11,20 @@
     vmtools_log_dir: "/tmp/vmware-tools"
   when: vmtools_log_dir is undefined or not vmtools_log_dir
 
-# VMware Tools config file on Linux or FreeBSD with open-vm-tools eariler than 10.1.x
+# VMware Tools config file on Linux
 - name: "Set fact of VMware Tools config file for {{ guest_os_ansible_distribution }}"
   ansible.builtin.set_fact:
     vmtools_config_file: "/etc/vmware-tools/tools.conf"
-  when: >-
-    guest_os_ansible_distribution != 'FreeBSD' or
-    vmtools_version is version('10.1.0', '<')
+  when: guest_os_ansible_distribution != 'FreeBSD'
 
-# VMware Tools config file on FreeBSD with open-vm-tools 10.1.x or later
+# VMware Tools config file on FreeBSD
 - name: "Set fact of VMware Tools config file for FreeBSD"
   ansible.builtin.set_fact:
-    vmtools_config_file: "/usr/local/share/vmware-tools/tools.conf"
-  when:
-    - guest_os_ansible_distribution == 'FreeBSD'
-    - vmtools_version is version('10.1.0', '>=')
+    vmtools_config_file: |-
+      {%- if vmtools_install_type is defined and vmtools_install_type == 'source' -%}/etc/vmware-tools/tools.conf
```

Comment:
Here doesn't need to check `vmtools_install_type is defined` as it is defined with `vmtools_version` in the same task file. And there could be 3 possible values for it: unknown, source, and package. Only when `vmtools_install_type == 'package'` we can assure its tools conf is `/usr/local/share/vmware-tools/tools.conf`. For the other two values, we can assume it is `/etc/vmware-tools/tools.conf`.